### ISSUE TITLE
TypeError: __init__() got an unexpected keyword argument 'wal_autocheckpoint'

### ISSUE THREAD

**Issue by ZahirSaleh** [ppetr/arduino-influxdb#2]
Hello,

When I run this script, I get the error 
```
==============================
Traceback (most recent call last):
  File "collect.py", line 124, in <module>
    main()
  File "collect.py", line 113, in main
    args.wal_autocheckpoint) as queue:
TypeError: __init__() got an unexpected keyword argument 'wal_autocheckpoint'
==============================
```
What could be wrong? Please help.

Zahir

**Comment by ppetr** [comment]
Thank you for reporting the issue. I forgot to upload the newer version of [persistent-queue-log](https://github.com/ppetr/persistent-queue-log). Meanwhile you can just remove `, wal_autocheckpoint=args.wal_autocheckpoint` from the list of arguments to `Queue` in `main` in collect.py.

**Comment by ppetr** [comment]
Fixed in _persistent-queue-log_ [9055372](https://github.com/ppetr/persistent-queue-log/commit/90553729dd2a536615a41ff6eb8daf86b483949a).
